Testreglar/4.1.2/Nett/nett-4.1.2b.json

```diff
@@ -0,0 +1,314 @@
+{
+	"namn": "Nett-4.1.2b For knapper kan tilgjengelig navn, rolle og tilstand bestemmes programmatisk 2023",
+	"id": "nett-4.1.2b",
+	"testlabId": 444,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nb",
+	"kravTilSamsvar": "<p>Navn og rolle:</p>\r\n<ul>\r\n<li>knapper har et tilgjengelig navn, som beskriver formålet med den aktuelle knappen og</li>\r\n<li>knapper har riktig rolle, som identifiserer funksjonen til den aktuelle knappen </li>\r\n</ul>\r\n<p>Tilstander, egenskaper og verdier:</p>\r\n<ul>\r\n<li>når tilstander, egenskaper og verdier ved knappen kan angis av brukeren, skal denne informasjonen også angis programmatisk og</li>\r\n<li>varsel om endringer i den aktuelle knappen er tilgjengelig for brukeragenter </li>\r\n</ul>",
+	"side": "2.1",
+	"element": "3.1",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		},
+		{
+			"title": "3.5"
+		},
+		{
+			"title": "3.6"
+		},
+		{
+			"title": "3.7"
+		},
+		{
+			"title": "3.8"
+		},
+		{
+			"title": "3.9"
+		},
+		{
+			"title": "3.10"
+		},
+		{
+			"title": "3.11"
+		}
+	],
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken side tester du?",
+			"ht": "<p>Angi URL eller side-ID.</p>",
+			"type": "tekst",
+			"label": "URL/Side:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Gir verktøyet resultater under \"Button has accessible name\" og/eller \"Image button has accessible name\"?",
```

Comment:
Hjelpeteksten her var rotete. For meg som aldri har brukt QW extension og har god teknisk kompetanse så gikk det ikke å følge instruksjonen.  Tror det enkleste er å gå tilbake til manuell sjekk/sjekke tilgjengelighetstreet. I fremtiden skal semi-automatiske regler legges i egen chrome extension uansett så dette er ikke robust for fremtiden.

Reply:
Vi oppdateres testregelen og lager den uten QW.